src/persist/EpochSys.hpp

```diff
@@ -453,7 +457,9 @@ T* EpochSys::register_alloc_pblk(T* b, uint64_t c){
     }
     PBlk* blk = b;
     blk->epoch = c;
-    assert(blk->blktype == INIT || blk->blktype == OWNED);
+    // Wentao: It's possible that payload is registered multiple times
+    assert(blk->blktype == INIT || blk->blktype == OWNED || 
+           blk->blktype == ALLOC); 
```

Comment:
This is because BEGIN_OP may be called more than once, and thus the same payload would be registered more than once. I don't think it is _always_ OK, but at least in some cases it's benign